frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -97,14 +104,14 @@ export function Breadcrumbs({
       : `/depositions/${previousDepositionId}?${previousSingleDepositionParams}`
 
   const chevronIcon = (
-    <SmallChevronRightIcon className="w-[8px] h-[8px] shrink-0" />
+    <SmallChevronRightIcon className={`w-[8px] h-[8px] shrink-0 ${type === "breadcrumb-light" ? "fill-[#999]" : "fill-black"}`} />
   )
 
   const plausible = usePlausible()
 
   return (
     <div
-      className="flex flex-col flex-auto gap-1"
+      className="flex flex-col flex-auto gap-1 max-w-xl"
```

Comment:
This should be linked to the variant imo

Reply:
Yes, you're right
So now i check if variant is "neuroglancer" and if yes we apply `max-w-xl `style